src/callback_bridge.h

```diff
@@ -2,30 +2,32 @@
 #define CALLBACK_BRIDGE_H
 
 #include <vector>
-#include <nan.h>
 #include <algorithm>
 #include <uv.h>
```

Comment:
Including `<uv.h>` breaks guarantees of ABI stability offered by N-API. https://github.com/sass/node-sass/pull/2312#issuecomment-552043349